data/map planets.txt

```diff
@@ -187,9 +190,10 @@ planet "New Guangdong"
 		fleet "Large Republic" 20
 
 planet Mikhailnov
+	attributes "near earth" factory urban
 	landscape land/ncity3
-	description "Mikhailnov was the second planet colonized by humanity before the Hyperdrive was invented. As the nations of earth battle for supremacy the Russian Empire wanted a colony to their name so the Ruler Gregory Mikhail funded with its own money the expedition to Magna Comae unfortunately Mikhail wouldnt survive the journey, and when settlers arrived they gave the planet his name to honor its deeds. "
-	description "    Over the years Mikhailnov became an industrial world large enough that they started to build their own ships outside Earth, and so Yatsunov Shipyards was formed as a subsidiary of the Colony ship constructor eventually absorbing it and becoming the first Interstellar shipyard outside Earth."
+	description "Mikhailnov was the second planet colonized by humanity before the hyperdrive was invented. As the nations of earth battle for supremacy the Russian Empire wanted a colony to their name so the Ruler Gregory Mikhail funded with its own money the expedition to Magna Comae unfortunately Mikhail wouldnt survive the journey, and when settlers arrived they gave the planet his name to honor its deeds."
+	description "	Over the years Mikhailnov became an industrial world large enough that they started to build their own ships outside Earth, and so Yatsunov Shipyards was formed as a subsidiary of the Colony ship constructor eventually absorbing it and becoming the first Interstellar shipyard outside Earth."
```

Comment:
Gregory Mikhail was a leader of Russia at some point, he along with Alexander Yatsunov masterminded the expedition to Magna Comae. Their colony ship stayed in orbit and expanded to a space station (soon).
Almost all outfits in human space are sold in the station and Yatsunov houses their main shipyards there, During the Turf Wars they had multiple conflicts with a recurring corporation: Nova Centauri (will become relevant later).

Reply:
Mikhailnov spaceport is a hole with lifts basically, at the bottom theres a monument to human space flight. From Gagarin to modern times.